.github/workflows/update-screenshots.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           yarn test:integrationUpdateScreenshots
 
       - name: Auto-commit updated screenshots
-        if: steps.check-update-screenshots.outputs.result == 'true'
+        if: always() && steps.check-update-screenshots.outputs.result == 'true'
```

Comment:
Just checking if these are meant to be wrapped in {{}} ? 
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#example-of-always

Reply:
I think it's okay as-is. We don't use the braces for our `skip-ci`.
https://github.com/Vonage/vonage-video-react-app/blob/8a0f5efb732863c33650ec55566b027dc0f4ab48/.github/workflows/run-tests.yml#L113